sigle/src/modules/analytics/Pagination.tsx

```diff
@@ -0,0 +1,55 @@
+import { ChevronLeftIcon, ChevronRightIcon } from '@radix-ui/react-icons';
+import { SubsetStory } from '../../types';
+import { Flex, IconButton } from '../../ui';
+
+interface PaginationProps {
+  currentPage: number;
+  onPageChange: (page: number) => void;
+  stories: SubsetStory[];
+  currentStoryItems: SubsetStory[];
```

Comment:
To make is more like a generic component that can later be reused, you could remove these props and just add a boolean `hasNextPage` for example, that way you can reuse it for referrers and other :)